spring-rabbit/src/main/java/org/springframework/amqp/rabbit/connection/ConnectionFactoryUtils.java

```diff
@@ -43,6 +44,13 @@
  */
 public final class ConnectionFactoryUtils {
 
+	private static final boolean USING_WEBFLUX;
+
+	static {
+		USING_WEBFLUX = ClassUtils.isPresent("org.springframework.web.reactive.function.client.WebClient",
```

Comment:
According to the AOT requirements and to avoid extra reflection info and execution ticks at runtime, this kind of constants should be named with a `_PRESENT` prefix.

I'm not sure also why you need this extra `static` block.
In my code I have it like this:
```
public static final boolean WEB_FLUX_PRESENT =
			ClassUtils.isPresent("org.springframework.web.reactive.DispatcherHandler", null);
```